sdk/core/core-amqp/src/cbs.ts

```diff
@@ -190,6 +190,10 @@ export class CbsClient {
     tokenType: TokenType
   ): Promise<CbsResponse> {
     try {
+      if (!this._cbsSenderReceiverLink) {
+        throw new Error("Attempted to negotiate a claim but a CBS link does not exist.");
```

Comment:
Also add a log?
`logger.verbose("[%s] Attempted to negotiate a claim but the CBS link does not exist.", this.connection.id);`

Reply:
The logging will occur in the catch statement below.

Reply:
I see.